HerPublicWebsite/Views/Partials/LocalAuthorityMessages/Confirmation/Pending.cshtml

```diff
@@ -0,0 +1,10 @@
+@model HerPublicWebsite.Models.Questionnaire.ConfirmationViewModel
+
+<p class="govuk-body">
+    <strong>@Model.LocalAuthorityName</strong> is not currently signed up to use this service yet. 
+    <strong>@Model.LocalAuthorityName</strong> or their official contractor will process your details and contact you if they need any more 
+    information on the contact details you provided when they have signed up to the service.
```

Comment:
"signed up to the service" → "signed up to use the service"
(Transcription error by me.)